c7n/resources/rdscluster.py

```diff
@@ -72,30 +72,13 @@ def process(self, clusters):
                         'Deleted RDS instance: %s',
                         instance['DBInstanceIdentifier'])
 
-            params = {'DBClusterIdentifier': cluster['DBClusterIdentifier']}
```

Comment:
why was this removed? afaics the `delete` action is no longer functional with this change